qiskit/ignis/verification/entanglement/parallelize.py

```diff
@@ -0,0 +1,449 @@
+
+
+'''
```

Comment:
You moved this comment to __init__.py; did you leave it here on purpose?

Reply:
In general yes, I want whoever reads it to be aware when it's time to move to Terra. I changed the wordings here to be specific to this file now